netlify.toml

```diff
@@ -0,0 +1,9 @@
+[build]
+  command = "npm run build"
+  functions = "netlify/functions/"
+  publish = "build/"
+
+[[redirects]]
+  from = "/admin"
+  to = "/index.html"
+  status = 301
```

Comment:
Can you add a comment explaining a bit more about how this works? I don't quite understand it.

Reply:
@kernfeld-cockroach I updated the redirect and added a comment.
It is also deployed here if you want to try:
- https://super-cool-site-by-galeran-prisma-io-77c50.netlify.app/
- https://super-cool-site-by-galeran-prisma-io-77c50.netlify.app/admin

Reply:
When I go to https://super-cool-site-by-galeran-prisma-io-77c50.netlify.app/whatever I just see a blank page. Is that the desired behavior?

Reply:
Yes, it's expected at the moment since there is no handling of non-matching routes on the React Router level here:
https://github.com/cockroachdb/cockroachdb-typescript/blob/master/src/App.tsx#L36-L39
Only 2 routes are defined, anything else will load the React app but do nothing (and page is then white). 

It could be customized with 
```
          <Route path="*">
            <NoMatch />
          </Route>
```
and
```
function NoMatch() {
  let location = useLocation();

  return (
    <div>
      <h3>
        No match for <code>{location.pathname}</code>
      </h3>
    </div>
  );
}
```
https://v5.reactrouter.com/web/example/no-match

For example

Reply:
Okay, thanks for the explanation.